src/main/java/org/gaul/s3proxy/Main.java

```diff
@@ -240,6 +240,13 @@ private static BlobStore parseMiddlewareProperties(BlobStore blobStore,
             blobStore = ReadOnlyBlobStore.newReadOnlyBlobStore(blobStore);
         }
 
+        String requiredContentMD5BlobStore = properties.getProperty(
+                S3ProxyConstants.PROPERTY_READ_ONLY_BLOBSTORE);
```

Comment:
Should this be `PROPERTY_REQUIRED_CONTENT_MD5_BLOBSTORE`?

Reply:
Done.